operations/iuf/IUF.md

```diff
@@ -632,6 +641,29 @@ HPE provides management NCN and managed node `sat bootprep` configuration files
 CFS configuration, image, and BOS session template definitions. The administrator may customize the files as needed. The files include
 variables, and the values used are provided by the recipe variables and/or site variables files specified when running `iuf run`.
 
+## Re-executing stages of an IUF session
+
+It is possible to re-execute stages of an IUF session by specifying `iuf run` with the desired stage and other `iuf` arguments. If no changes were made to the product distribution files in the media directory, `iuf run` will
+re-execute any Argo steps that failed during the previous invocation of `iuf run`. Any Argo steps that previously executed successfully will be skipped if possible. If the `-f` argument is specified, all Argo steps will be
+re-executed, regardless of whether they succeeded or failed during the previous invocation of `iuf run`.
+
+### Removing a product from an IUF session
+
+If the administrator wants to remove a product from the IUF session, they must re-execute `iuf run` for the `process-media` stage with the product distribution file and uncompressed content removed from the media directory. This
+removes knowledge of that product from the existing IUF activity.
+
+If any previously-executed stages performed operations with that product, it may be necessary to re-execute them as well to remove artifacts or metadata related to the product, e.g. to remove CFS configuration layers and rebuild
+images without that product present.
+
+### Adding a product to an IUF session
+
+If the administrator wants to add a new product (or new version of an existing product) to the IUF session, they must re-execute `iuf run` for the `process-media` stage with the new product distribution file added to the media
```

Comment:
Try to remove any conversational explanation for procedures, like "If the administrator wants to add..." If this is an optional procedure, append (Optional) to the topic title.

I am wondering why these aren't written as procedures with commands and code blocks??

Suggested updates:

To add a new product (or new version of an existing product) to the IUF session, re-execute `iuf run` for the `process-media` stage with the new product distribution file added to the media directory. This adds knowledge of that product to the existing IUF activity. If the new product is being used in place of a different version of the product, remove the previous version of the product distribution file and uncompressed content from the media directory at the same time the new version is added.

If previously executed stages performed operations using the old product version, re-execute them to remove artifacts or metadata related to the old version of the product, e.g. to remove CFS configuration layers and rebuild images to ensure only the new version of the product is present.

Reply:
Fixed in next commit.

The entire IUF.md file is just informational, not exact steps to run as part of an install or upgrade.  I do like your "optional" note, but I think that would be for the install/upgrade workflow part only.  I will review that portion of the doc for the next Jira and PR and look for any spots that could use an optional tag.

I don't mind adding a separate example of this in a later Jira and PR (it will take a bit and I'd like to get this content in for the next round of installs starting up).